Update update status (#871)
meilisearch/documentation: learn/advanced/asynchronous_updates.md

You can use the `status` field returned by [the update route](/reference/api/updates.md) to determine if a process has been committed to MeiliSearch or not.

- status: `enqueued` => Not yet begun. If MeiliSearch is killed and then restarted, the task will remain enqueued and be processed eventually.

> - status `in_progress` => Processing. If MeiliSearch is killed, there will be no consequences, since no part of the task has been committed to MeiliSearch. After restarting, Meilisearch will treat the task as `enqueued`.

Reviewer: I'm confused; isn't `in_progress` a status for Dumps, not updates?

According to your first comment, the three statuses described on this page should be `enqueued`, `processed`, and `failed`.

Reviewer: ```suggestion
- status `failed` => Not successful. If MeiliSearch is killed, there will be no consequences, since no part of the task has been committed to MeiliSearch.
```

Author: I thought you talked about dumps AND updates in this paragraph. Do you want to only mention the updates?

Author: The paragraph does not list the status but what happen if you kill meilisearch

Reviewer: Good point, it's really unclear. I thought it was only speaking about updates, but clearly not since there is no `done` status for updates.

Reviewer: BTW, I found where it mentions the possible values of `status`: https://docs.meilisearch.com/learn/advanced/asynchronous_updates.html#understanding-updates

Reviewer: In this case, right above this list it says "You can use the status field returned by the update route to determine if a process has been committed to MeiliSearch or not." Since the update route doesn't work for Dumps, I think it should just be updates in this paragraph, after all.

Author: What should I do then? 🙂
Would you rather I open an issue?

Reviewer: Yes, thanks! :)

Author: #877
